### PR TITLE
Add vdirsyncer (+ request_toolbelt, atomicwrites python packages)

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Hi,

this PR adds the following python libraries:
- requests_toolbelt
- atomicwrites

To be able to add
- vdirsyncer

A syncing application for the vdir storage format which can be used to sync ownCloud calendar and contacs (vcf, ical files).

~~It is WIP, though it builds on my machine. I'd like to clean up the history before merging!~~

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
I'm ready with this, tested it on my testing machine. Do I have an "ack" for squashing?

File: pkgs/tools/misc/vdirsyncer/default.nix [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  name = "vdirsyncer-${version}";
+  namePrefix = "";
+
+  src = fetchurl {
+    url = https://github.com/untitaker/vdirsyncer/archive/0.4.2.tar.gz;
+    sha256 = "008181mglbrp5jsvpxr59b6w4mw26h4s4gwij152i47mfbrizsl4";
+  };
+
+  pythonPath = with pythonPackages; [
+                icalendar
```
Comment (aszlig) [inline pkgs/tools/misc/vdirsyncer/default.nix]: Please indent with two spaces, like:

```nix
{
  pythonPath = with pythonPackages; [
    icalendar
    foo
    bar
  ];
}
```

File: pkgs/tools/misc/vdirsyncer/default.nix [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                # python-setuptools
+                # python-pytest (check)
+                # python-pytest-localserver (check)
+                # python-pytest-xprocess (check)
+                # python-werkzeug-git (check)
+                # python-wsgi-intercept (check)
+                # radicale (check)
+                ];
+
+  buildInputs = [ ];
```
Comment (aszlig) [inline pkgs/tools/misc/vdirsyncer/default.nix]: It's [empty by default](https://github.com/NixOS/nixpkgs/blob/73c43b4e4f9d47e4a47fef6b210fb97a1ad8cc75/pkgs/development/python-modules/generic/default.nix#L13), so you can just leave it out.

**Comment by aszlig** [comment]
Squashing hereby *ack*ed :-)

Btw. you can use commit messages such as `fixup! message` or `squash! message` and if you have `rebase.autosquash` set to `true` it will automatically apply the corresponding actions to the commits in your next rebase.

**Comment by matthiasbeyer** [comment]
> Btw. you can use commit messages such as fixup! message or squash! message and if you have rebase.autosquash set to true it will automatically apply the corresponding actions to the commits in your next rebase.

Ah, thanks for this one! Unfortunately I do not do the fixups right after the commits, so I have to manually rebase because I have to reorder the commits. But anyways, good to know!

**Comment by matthiasbeyer** [comment]
@aszlig Squashed into three commits, each for one package which was added by this PR.

**Pull request merged**